components/cosmetic_filters/renderer/cosmetic_filters_js_handler.h

```diff
@@ -39,7 +39,8 @@ class CosmeticFiltersJSHandler {
   // filtering is enabled, and returns whether or not to proceed with cosmetic
   // filtering.
   bool ProcessURL(const GURL& url, std::optional<base::OnceClosure> callback);
-  void ApplyRules(bool de_amp_enabled);
+  void ApplyRules(bool de_amp_enabled,
+                  bool is_youtube_hd_quality_playback_enabled);
```

Comment:
this is definitely not what we want to do here, we should selectively enable these scripts on the browser side by using different data sources, but also right now this is going to be disabled when adblock is disabled which is clearly not what we want here